code-studio/src/js/components/progress/course_progress_row.jsx

```diff
@@ -2,29 +2,47 @@
 import React from 'react';
 import { STAGE_TYPE } from './types';
 import StageProgress from './stage_progress.jsx';
+import color from '../../color';
+
+const STYLES = {
```

Comment:
Ah you're totally right, the Airbnb style guide we reference recommends [lowercase `const` names](https://github.com/airbnb/javascript#references--prefer-const).